Neo4j Tutorials.txt

```diff
@@ -0,0 +1,13 @@
+Getting Started: https://neo4j.com/developer/get-started/
```

Comment:
I don't think it's necessary to have this in the repo

Reply:
that one link or the tutorials?

Reply:
The entire file doesn't need to be in the repo